### PR TITLE
[RFR] Korean Translation

### PR DESCRIPTION
**Pull request by kimdhoe**

### CODE REVIEW

File: ko/_syntax.md [hunk 1]
```diff
@@ -0,0 +1,1314 @@
...
+// Nope
+$font-stack: ('Helvetica', 'Arial', sans-serif)
+
+// Nope
+$font-stack: ('Helvetica', 'Arial', sans-serif,)
+{% endhighlight %}
+  </div>
+</div>
+
+리스트에 새로운 아이템을 추가할 때는, 언제나 제공된 API를 이용하세요. 수동���로 새로운 아이템을 추가하려고 하지 마세요.
```
Comment (HugoGiraudel) [inline ko/_syntax.md]: I notice `���` here. Is it intended? Only on my side perhaps?

**Comment by HugoGiraudel** [comment]
This looks great to me, except for the comment I made. :)

**Comment by kimdhoe** [comment]
Not intended definitely. I'll fix it right away.

**Comment by HugoGiraudel** [comment]
Thanks. I have not carefully reviewed the whole thing so there might be some other occurrences of this.

**Comment by kimdhoe** [comment]
Right. I will review it once more.

**Comment by kimdhoe** [comment]
OK! Hopefully everything is good this time.

**Comment by HugoGiraudel** [comment]
Terrific: http://sass-guidelin.es/ko/! Thank you so much. :)

**Comment by kimdhoe** [comment]
It is my pleasure to do :) Thanks!
I just realized that `href`s in hash anchors also should be translated. Links in toc are all broken. I will deal with it tomorrow.

**Pull request merged**